<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LobbyScene</class>
 <widget class="QWidget" name="LobbyScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lobby</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="ticketCountLabel">
       <property name="text">
        <string>Ticket Count:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ticketCount">
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="characterCountLabel">
       <property name="text">
        <string>Character Count:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="characterCount">
       <property name="text">
        <string>0/20</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="characterList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="startGame">
       <property name="text">
        <string>Start Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createCharacter">
       <property name="text">
        <string>Create Character</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="purchaseTicket">
       <property name="text">
        <string>Purchase Ticket</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteCharacter">
       <property name="text">
        <string>Delete Character</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitGame">
       <property name="text">
        <string>Exit Game</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lastLogin">
     <property name="text">
      <string>Last Login: 2019/01/31 22:08</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
